### DISCUSSION TITLE
Auto Reload

### DISCUSSION THREAD

**Discussion by leiweibau** [netalertx/NetAlertX#110] [Ideas]
hello,

in my Fork i implement a solution, for auto reloading, when i reenter the page. Maybe you will find it interesting.

```
document.addEventListener("visibilitychange",()=>{
   if(document.visibilityState==="visible"){
       window.location.href = window.location.href.split('#')[0];
   }
})
```

With best regards

https://github.com/jokob-sk/Pi.Alert/blob/f7fbf9488a9605ded4821149d50af04e5858a6d8/front/php/templates/header.php#L119

**Comment by jokob-sk** [comment]
Thanks, much appreciated! implemented :)
